common/src/main/java/com/hortonworks/streamline/common/SchemaValueConverter.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2017 Hortonworks.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *   http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package com.hortonworks.streamline.common;
+
+import com.hortonworks.registries.common.Schema;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
+
+public final class SchemaValueConverter {
+    private SchemaValueConverter() {
+    }
+
+    public static Map<String, Object> convertMap(Schema schema, Map<String, Object> value) {
```

Comment:
I think this is a nice functionality and can be reused in other places. Can you add some java doc on what this method does?